src/Management/test/Endpoint.Test/Services/EndpointMiddlewareTest.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
There should be test coverage for emitting a `?` suffix when optional constructor parameters are used.